persistence-cassandra/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/cassandra/testkit/TestUtil.scala

```diff
@@ -9,7 +9,7 @@ import com.typesafe.config.{ Config, ConfigFactory }
 
 object TestUtil extends AbstractTestUtil {
 
-  def persistenceConfig(testName: String, cassandraPort: Int, useServiceLocator: Boolean) = ConfigFactory.parseString(s"""
+  def persistenceConfig(testName: String, cassandraPort: Int, useServiceLocator: Boolean, offsetStoreAutoCreate: Boolean = true) = ConfigFactory.parseString(s"""
```

Comment:
I don't think this is really needed - there's lots of other arguments that could be added here, such as whether the journal tables should be auto created, but they aren't, because it's quite trivial to disable those.  I think this method should just produce config that's not trivial, so for example the cassandra port needs to be passed to three different places in the config, that's why it's taken here.

So I would say the changes to this file should be reverted.